app/views/payments/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title, "Betalingen - SOFIA" %>
+
+<div class="container footer-padding">
+  <div class="row pt-4">
+    <div class="col-sm-12 d-flex justify-content-between align-items-center">
+      <h1>Betalingen</h1>
+    </div>
+
+    <table class="table table-striped mt-2">
+      <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Status</th>
+        <th scope="col">Bedrag</th>
+        <th scope="col">Gebruiker</th>
+      </tr>
+      </thead>
+      <tbody>
+      <% @payments.each do |payment| %>
+        <tr>
+          <th scope="row"><%= payment.id %></th>
+          <td><%= t(payment.status) %></td>
```

Comment:
Maybe make the first letter uppercase? Minor details, but looks a bit nicer to me